### DISCUSSION TITLE
No macos version builds anymore?

### DISCUSSION THREAD

**Discussion by bjesuiter** [seaweedfs/seaweedfs#3152] [General]
From Version > [2.97](https://github.com/chrislusf/seaweedfs/releases/tag/2.97) there are no darwin zips in the release assets. 
Is that planned or a bug? 

Because I would still like to use the darwin versions and would prefer not needing to build it myself.

**Comment by chrislusf** [comment]
The macos build fails https://github.com/chrislusf/seaweedfs/actions/runs/2445202224

Does not happen locally. Does it happen for you? Not quite sure what is the problem.

**Reply by bjesuiter** [reply]
Thanks for fast answer! Hope you figure it out soon, I think I'm not knowledgeable enough in Go and with the project to help searching for the error :/

**Comment by dyus** [comment]
I make a little research and figure out that problem with macos build is related to go 1.18 and cgo in package go-ieproxy.
I reproduce it locally with flag  CGO_ENABLED=0  which is enabled in ci pipeline.
Possible fixes are: downgrade version of go-ieproxy to v0.0.1 or replace flag CGO_ENABLED=0 with CGO_CPPFLAGS="-Wno-error -Wno-nullability-completeness -Wno-expansion-to-defined -Wno-builtin-requires-header". I'm not good enough in go to proof are this fixes good at all. For example downgrade of go-ieproxy force downgrade of other packages
```
go: downgraded github.com/mattn/go-ieproxy v0.0.3 => v0.0.1
go: downgraded gocloud.dev v0.25.0 => v0.24.0
go: downgraded gocloud.dev/pubsub/natspubsub v0.25.0 => v0.24.0
go: downgraded gocloud.dev/pubsub/rabbitpubsub v0.25.0 => v0.24.0
```
The other hand enable cgo and ignore some warning sounds bad.

related issues:
- https://github.com/Azure/azure-pipeline-go/issues/36
- https://github.com/mattn/go-ieproxy/issues/34

**Reply by chrislusf** [reply]
@dyus Thanks for your findings! I was waiting ieproxy to fix this problem, but the fix https://github.com/mattn/go-ieproxy/issues/34 seems stuck.

I am setting `CGO_ENABLED=1` and see how it goes for the next build.
